alfresco-transform-core-aio/alfresco-transform-core-aio-boot/src/main/java/org/alfresco/transformer/AIOCustomConfig.java

```diff
@@ -32,13 +32,30 @@
 import org.alfresco.transformer.transformers.MiscAdapter;
 import org.alfresco.transformer.transformers.PdfRendererAdapter;
 import org.alfresco.transformer.transformers.TikaAdapter;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Primary;
 
 @Configuration
 public class AIOCustomConfig
 {
+    @Value("${libreoffice.executor.path}")
+    private String libreofficePath;
+
+    @Value("${pdfrenderer.executor.path}")
+    private String pdfRendererPath;
+
+    @Value("${imagemagick.executor.path.exe}")
+    private String EXE;

```

Comment:
Can we change these to camel case variable names which are a little more descriptive